services/m3aggregator/config/aggregator.go

```diff
@@ -776,20 +655,12 @@ func (b jitterBucketsByIntervalAscending) Less(i, j int) bool {
 	return b[i].FlushInterval < b[j].FlushInterval
 }
 
-// timerQuantileSuffixFnType is the timer quantile suffix function type.
```

Comment:
Do we no longer allow users to customize the quantile suffix functions by providing a different type?

Reply:
We don't have another suffix type for user to config to anyway, we need a code change to add the alternative suffix func we could add the config in that diff, also the config should be in m3metric.

Reply:
Sure sounds good.